ci/recipe/meta.yaml

```
@@ -20,6 +20,7 @@ requirements:
     - python {{ python }}
     - biom-format {{ biom_format }}
     - bioconductor-dada2 1.22.0
+    - r-base >=4.2.0
```

```suggestion
    - r-base {{ r_base }}
```

We should set this via `conda_build_config` so that we don't have to update this everywhere each time there is a new bioconductor